todo.c

```diff
@@ -45,21 +45,27 @@ void todo_swap(todo_t *a, todo_t *b) {
 }
 
 void todo_add_sort_option(enum sort_option opt) {
-	//sort_options_count++;
 	sort_options = realloc(sort_options, sizeof(int) * (++sort_options_count));
 	sort_options[sort_options_count - 1] = opt;
 }
 
+
+/*
+	I use bubble sort because it is one of the simplest,
+	even if it is very slow. Because, i could add custom
+	ordering rules when i know how the algorithm works.
+*/
 void todo_start_sort(todo_t **todos, size_t n) {
 	for (size_t i = 0; i < n-1; i++) {
 		for (size_t j = 0; j < n-i-1; j++) {
 			todo_t *a = todos[j];
 			todo_t *b = todos[j+1];
 
-			// Let me explain,
-			// mode = 1 = ASC
-			// mode = 0 = DESC
-			int mode = 1;
+			/*
+				Custom ordering rules, made by the user by
+				putting some command line flags.
+			*/
+			int isAscending = 1;
```

Comment:
Will it have only 2 values? maybe using `bool` would be better.

Reply:
What is the difference using `bool` instead of `int`? Are one of them more light (in bytes) than the other?